src/Kafka/ManageKafkaPermissions/components/SelectAccount.tsx

```diff
@@ -0,0 +1,187 @@
+      validated: ValidatedOptions.default,
+    });
+    setIsOpen(false);
+  };
+  const options = initialOptions.length
+    ? [
+        <SelectGroup key="all_accounts_group">
+          <SelectOption
+            key="*"
+            value="*"
+            description={t(
+              "manage_permissions_dialog.all_accounts_description"
+            )}
+          >
+            {t("manage_permissions_dialog.all_accounts_title")}
+          </SelectOption>
+        </SelectGroup>,
+        <Divider key="all_accounts_divider" />,
+        <SelectGroup
+          label={t(
+            "manage_permissions_dialog.all_accounts_service_account_group"
+          )}
+          key="service_accounts_group"
+        >
+          {initialOptions
+            .filter(
+              (principal) =>
+                principal.principalType === PrincipalType.ServiceAccount
+            )
+            .sort((a, b) =>
+              a.displayName && b.displayName
+                ? a.displayName.localeCompare(b.displayName)
+                : -1
+            )
+            .map((principal, index) => (
+              <SelectOption
+                key={index}
+                value={principal.id}
+                description={principal.displayName}
+              >
+                {principal.id}
+              </SelectOption>
+            ))}
+        </SelectGroup>,
+        <Divider key="user_account_divider" />,
+        <SelectGroup
+          label={t("manage_permissions_dialog.all_accounts_user_account_group")}
+          key="user_accounts_group"
+        >
+          {initialOptions
+            .filter(
+              (principal) =>
+                principal.principalType === PrincipalType.UserAccount
+            )
+            .map((principal, index) => (
+              <SelectOption
+                key={index}
+                value={principal}
+                description={principal.displayName}
+              >
+                {principal.id}
+              </SelectOption>
+            ))}
+        </SelectGroup>,
+      ]
+    : [];
+
+  const customFilter = (
+    _: React.ChangeEvent<HTMLInputElement> | null,
+    value: string
+  ) => {
+    if (!value) {
+      return options;
+    }
+
+    const input = new RegExp(value, "i");
+    return options
+      .filter((accounts) => Array.isArray(accounts.props.children))
+      .map((account) =>
+        account.props.children.filter(
+          (allAccounts: { props: { value: string; description: string } }) =>
+            input.test(allAccounts.props.value) ||
+            input.test(allAccounts.props.description)
+        )
+      );
+  };
+
+  return (
+    <FormGroupWithPopover
+      labelHead={t("manage_permissions_dialog.account_id_title")}
+      fieldId="kafka-instance-name"
+      fieldLabel={t("manage_permissions_dialog.account_id_title")}
+      labelBody={t("manage_permissions_dialog.account_id_help")}
+      buttonAriaLabel={t("manage_permissions_dialog.account_id_aria")}
```

Comment:
I know this text string isn't defined in this PR this is defined, but when I look at the value being used for the Help button aria-label, it just says "The account"

If we follow the example in PF, then a better label for this button would be "More info for Account field"